tests/idris2/operators/operators001/Test.idr

```diff
@@ -6,6 +6,9 @@ import Data.Vect
 private typebind infixr 0 =@
 private infixr 0 -@
 
+%hide Prelude.Ops.infixl.(|>)
+%hide Prelude.Ops.infixr.(|>)
```

Comment:
Without this line I would still get warnings, although that operator doesn't exist.